public/plgapp/plgapp.js

```diff
@@ -85,6 +85,10 @@ var PlgApp = function () {
         });
     };
 
+    this.registerSessionTimeoutCallback = function(timeout_callback, seconds_before_timeout) {
```

Comment:
Line is too long.<br>Identifier 'timeout_callback' is not in camel case.<br>Identifier 'seconds_before_timeout' is not in camel case.